### ISSUE TITLE
nemo seahorse build does not support latest gnupg

### ISSUE THREAD

**Issue by jean-christophe-manciot** [linuxmint/nemo-extensions#237]
Ubuntu 17.04
seahorse 3.4.1
gnupg 2.2.1
libgcrypt 1.7.9

```
nemo-seahorse# ./configure --build=x86_64-linux-gnu --prefix=/usr --includedir=\${prefix}/include --mandir=\${prefix}/share/man --infodir=\${prefix}/share/info --sysconfdir=/etc --localstatedir=/var --disable-silent-rules --libdir=\${prefix}/lib/x86_64-linux-gnu --libexecdir=\${prefix}/lib/x86_64-linux-gnu --disable-maintainer-mode --disable-dependency-tracking
...
checking for gpg... /usr/bin/gpg
checking for appropriate GnuPG version... configure: error: Appropriate version of GnuPG not found. Please install one of versions: 1.2 1.4 2.0 2.1
```

**Comment by nullptrT** [comment]
`sed -i "s/1.2 1.4 2.0/1.2 1.4 2.0 2.1 2.2/" configure.ac` patches this problem for at least until gnupg v2.2.

**Comment by jean-christophe-manciot** [comment]
> Finally fixed, for now, with [fdf7351](https://github.com/linuxmint/nemo-extensions/commit/fdf7351b9b1ce2932748613b563c9a64249ed5ef).

I could successfully build nemo-seahorse-4.0.1 on **Debian buster**.
It will most probably be also successful on Ubuntu disco.
